ddb_single/query.py

```
@@ -62,20 +62,20 @@ def search(self, *queries, pk_only=False, limit=None):
-        Get single item.
+        アイテムを取得
         Args:
             pk: Primary key
         """
         res = self.__table__.get_item(pk)
         return res
 
-    # ユニークキーで検索
-    def get_by_unique(self, value, pk_only=False, keys: List[str | DBField] = []):
+    def get_by_unique(
+        self, value, pk_only=False, keys: List[str | DBField] = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)